Source/Outposts/Outposts.csproj

```
@@ -86,8 +84,13 @@
     <Compile Include="WITabs\WITab_Outpost_Health.cs" />
     <Compile Include="WITabs\WITab_Outpost_Items.cs" />
     <Compile Include="WITabs\WITab_Outpost_Needs.cs" />
+    <Compile Include="WorldComponent_OutpostAttacked.cs" />
+  </ItemGroup>
+  <ItemGroup>
+    <PackageReference Include="Lib.Harmony">
+      <Version>2.2.2</Version>
+    </PackageReference>
```

```suggestion
    <PackageReference Include="Lib.Harmony">
      <Version>2.2.2</Version>
      <ExcludeAssets>runtime;</ExcludeAssets>
    </PackageReference>
```
This just ensures the `0Harmony` dll doesn't end up in the mod by accident.